### ISSUE TITLE
Hovering over cards (in both your and your opponent's deck) does not always show the card

### ISSUE THREAD

**Issue by xSnowLeopardx** [HearthSim/Hearthstone-Deck-Tracker#4593]
### Expected Behavior

Hovering over the card in your and your opponent's deck should _always_ show the card with all its info.

### Actual Behavior

In _some cases_, hovering over your or your opponent's deck does not show the card.

### Steps to reproduce behavior

No. It happens randomly. Closing and opening HDT does not help. Launching HDT in Administrator Mode does not help, either. There is a chance that it works the next game, but how long it works or does not work is not known.

### Log/Screenshots

[hdt_log.txt](https://github.com/user-attachments/files/17692964/hdt_log.txt)

**Comment by beheh** [comment]
Hi there! Could you be a little more specific? Does it not show anything? Does it show the placeholder? If so, is this maybe a duplicate of #4584?

**Comment by xSnowLeopardx** [comment]
Still curious what could be the problem. Is there a way for me to check what is happening myself?

**Comment by beheh** [comment]
You don't happen to have this setting disabled, do you?
![image](https://github.com/user-attachments/assets/776b7deb-b6c3-4e9d-9062-021b248de8b4)

**Comment by xSnowLeopardx** [comment]
> You don't happen to have this setting disabled, do you? ![image](https://private-user-images.githubusercontent.com/929769/387188001-776b7deb-b6c3-4e9d-9062-021b248de8b4.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzE5NTgyMTEsIm5iZiI6MTczMTk1NzkxMSwicGF0aCI6Ii85Mjk3NjkvMzg3MTg4MDAxLTc3NmI3ZGViLWI2YzMtNGU5ZC05MDYyLTAyMWIyNDhkZThiNC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMTE4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTExOFQxOTI1MTFaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hNDdiMWQzOTIzM2E4ZDZiOWE1MDY1ZTAyMjk3NjQ5YWY4Y2I5MWQ4ZmI1MjVmZjY4YmVlNmQ2Mzg4Y2ZhYTYwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.yl6KYFLGAW_5F9A6pk9WaELyq6Q3j9MG5YYCI04rk5o)
> 
> Although the random part is what I find especially weird.

Nope :/ :
![aaaaa](https://github.com/user-attachments/assets/64db13c3-6073-4d6c-98b2-01a4e71940f7)

**Comment by xSnowLeopardx** [comment]
Still happening...

**Comment by xSnowLeopardx** [comment]
Still happening...

**Comment by beheh** [comment]
We're currently working on refactoring card tooltips to a new system, so we're hoping this is fixed in an upcoming HDT version.

**Comment by xSnowLeopardx** [comment]
> We're currently working on refactoring card tooltips to a new system, so we're hoping this is fixed in an upcoming HDT version.

Hi again! Could you let me know if this possible fix has been pushed or not yet? I am still facing the problem.

**Comment by beheh** [comment]
No, it'll still be a few weeks and we're on break right now. Keep an eye on the Changelog.

**Comment by xSnowLeopardx** [comment]
> No, it'll still be a few weeks and we're on break right now. Keep an eye on the Changelog.

Thanks mate! Appreciate all the efforts.

**Comment by beheh** [comment]
Hello, we identified the root cause and have fixed this issue in the next HDT version, releasing some time this week. We believe it was related to having an E.T.C. Band in your deck, but having that part of the deck list disabled in the settings.

Until the update is released, you should be able to fix this by reenabling E.T.C.'s Band in the player deck list settings in HDT.

Thanks again for the report!

**Comment by xSnowLeopardx** [comment]
Hi @beheh , thank you for your response!

If you are talking about the following setting, it was always turned on for me:

![Image](https://github.com/user-attachments/assets/4f7c1c95-3a5c-4b1b-8c70-05bd141a9e10)

I ironically started playing and having E.T.C. in the last few weeks, but did not see any differences with the card in my deck vs. before. That setting was always on, so I am not sure what the problem really can be...

**Comment by xSnowLeopardx** [comment]
@beheh thanks! I can already tell from playing a few games that it is working well now!
